TACC/protx-dashboard
# task/WG-95: update resources DB to fix cooks hospital categories

## Overview: ##

Update resources db.  The new database now has some Cook Children's locations marked as 9999 (`FULL_NAICS_CODE`) in business_locations table.

## Related Jira tickets: ##

* [WG-95](https://jira.tacc.utexas.edu/browse/WG-95)

## Testing Steps: ##
1. Ensure that you see Cook Children's Locations as one of the "Resources" categories

## UI Photos:

before:
![Screenshot 2023-08-07 at 4 15 44 PM](https://github.com/TACC/protx-dashboard/assets/8287580/7a21cc71-3b53-4c37-840c-d6c29c7fbadb)

now:
![Screenshot 2023-08-08 at 10 59 59 AM](https://github.com/TACC/protx-dashboard/assets/8287580/308cfaf9-0d32-480a-b84d-d736316161a2)
